packages/web/src/pages/dashboard-page/components/AlbumsTab.tsx

```diff
@@ -0,0 +1,54 @@
+import { Nullable } from '@audius/common/utils'
+import { Paper } from '@audius/harmony'
+import { useSelector } from 'react-redux'
+
+import { TracksTable, TracksTableColumn } from 'components/tracks-table'
+
+import { makeGetDashboard } from '../store/selectors'
+
+import { showMoreLimit, tablePageSize } from './constants'
+import { useFilteredAlbumData } from './hooks'
+import { AlbumFilters } from './types'
+
+const albumTableColumns: TracksTableColumn[] = [
+  'spacer',
+  'trackName',
+  'releaseDate',
+  'reposts',
+  'overflowMenu'
+]
+
+type ArtistDashboardAlbumsTabProps = {
+  selectedFilter: Nullable<AlbumFilters>
+  filterText: string
+  onClickRow: (record: any) => void
+}
+
+export const ArtistDashboardAlbumsTab = ({
+  selectedFilter,
+  filterText,
+  onClickRow
+}: ArtistDashboardAlbumsTabProps) => {
+  const { account } = useSelector(makeGetDashboard())
+  const filteredData = useFilteredAlbumData({
+    selectedFilter,
+    filterText
+  })
+
+  if (!filteredData.length || !account) return null
```

Comment:
If I read this correctly, we render nothing at all if the search filter returns no results or no albums are returned from the selector?
I would have expected some kind of placeholder in these scenarios, as it might just look broken otherwise?

Reply:
hm good point. didn't see anything in the mocks specifically addressing this case, but i'll bring it up.